website/i18n/en.json

```diff
@@ -695,7 +695,7 @@
     "categories": {
       "Introduction": "Introduction",
       "Getting Started": "Getting Started",
-      "Legal-Tech Tutorials": "Legal-Tech Tutorials",
+      "Beginners Tutorials": "Beginners Tutorials",
```

Comment:
I understand the idea but I'm not sure I like `Beginners`. Maybe the distinction is between `Authoring` and `Developing` or `Users` vs `Developers`.

Reply:
changed to `User Tutorials`? :O